src/codegen.cpp

```diff
@@ -6345,7 +6345,7 @@ extern "C" void jl_init_codegen(void)
 #endif
 
 #ifdef LLVM36
-    EngineBuilder eb(std::move(std::unique_ptr<Module>(engine_module)));
+    EngineBuilder eb((std::unique_ptr<Module>(engine_module)));
```

Comment:
No; without the parentheses, `eb` becomes a function declaration instead of an object constructor call <https://en.wikipedia.org/wiki/Most_vexing_parse>.